dev.py

```diff
@@ -347,9 +343,8 @@ def run_tests(config, services, exclude, limit_to):
             info(f"Cannot run tests for {service}: {service} is not running.", "red")
             continue
         port = config.get_service_port(service)
-        version = config.get_wp_version(service)
         info(f"Running tests for {service} (port {port})")
-        cmd = f'/bin/bash -c "export WP_PORT={port} WP_VERSION={version}; composer run phpunit'
```

Comment:
We can get this from within the WordPress environment itself. Not sure why I didn't see that option when I first set this up.